lib/core/src/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobin.hs

```diff
@@ -881,22 +921,25 @@ makeChange minCoinFor requiredCost mExtraCoinSource inputBundles outputBundles
         -- asset count:
         & NE.zipWith (\m1 (m2, c) -> (m1 <> m2, c))
             changeForNonUserSpecifiedAssets
-        -- Finally, if there are any maps with excessive token quantities, then
+        -- Finally, if there are any maps that are oversized (in any way), then
         -- split these maps up along with their corresponding output coins:
-        & splitMapsWithExcessiveQuantities
+        & splitOversizedMaps
       where
-        splitMapsWithExcessiveQuantities
+        splitOversizedMaps
             :: NonEmpty (TokenMap, Coin) -> NonEmpty (TokenMap, Coin)
-        splitMapsWithExcessiveQuantities =
+        splitOversizedMaps =
             -- For the sake of convenience when splitting up change maps and
             -- output coins (which are treated as weights), treat each change
             -- map and its corresponding output coin as a token bundle.
             fmap unbundle . split . fmap bundle
           where
             bundle (m, c) = TokenBundle c m
             unbundle (TokenBundle c m) = (m, c)
-            split = flip equipartitionTokenBundlesWithMaxQuantity
-                maxTxOutTokenQuantity
+            split b = b
+                & flip splitBundlesWithExcessiveAssetCounts
+                    (tokenBundleSizeExceedsLimit assessBundleSize)
+                & flip splitBundlesWithExcessiveTokenQuantities
```

Comment:
In #2555 I ran into:
```
on\",\"message\":\"That's embarrassing. It looks like I've created an invalid transaction that could not be parsed by the node. Here's an error message that may help with debugging: HardForkApplyTxErrFromEra S (S (S (Z (WrapApplyTxErr {unwrapApplyTxErr = ApplyTxError [LedgerFailure (UtxowFailure (UtxoFailure (MaxTxSizeUTxO 4103 4096)))]}))))\"}")
```
while rebased to this branch.

I think this is due to `assignCoinsToChangeMaps` adding Coin values to the bundle _after_ we do the size check.

It would be nice to have `prop_performSelection` run with arbitrary size limits.

Reply:
Wait, I might have misunderstood the error above, although I also believe he reasoning should also hold 👀

Reply:
This should be it:
```haskell
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=64)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=65)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=66)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=67) FAILED [1]
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=68) FAILED [2]
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=69)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=70)

Failures:

  /Users/anviking/IOHK/cardano-wallet/lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs:747:13:
  1) API Specifications, SHELLEY_TRANSACTIONS, TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=67)
       expected: Status {statusCode = 202, statusMessage = "Accepted"}
        but got: Status {statusCode = 500, statusMessage = "Internal Server Error"}

       from the following response: Left (DecodeFailure "{\"code\":\"created_invalid_transaction\",\"message\":\"That's embarrassing. It looks like I've created an invalid transaction that could not be parsed by the node. Here's an error message that may help with debugging: HardForkApplyTxErrFromEra S (S (S (Z (WrapApplyTxErr {unwrapApplyTxErr = ApplyTxError [LedgerFailure (UtxowFailure (UtxoFailure (OutputTooBigUTxO [(Addr Mainnet (KeyHashObj (KeyHash \\\"f1946d14754ce4eb2a645aeff910bb279b88f0b3b69dc4f07cf28df5\\\")) (StakeRefBase (KeyHashObj (KeyHash \\\"509722367fde9ca9d9bdaee4c47353affff717df8f4fa82773ff828f\\\"))),Value 1997174061 (fromList [(PolicyID {policyID = ScriptHash \\\"4ff049585c4b3070563966370f5427d4a2f3588bce4146d57a93c7d3\\\"},fromList [(\\\"00000000000000000SeaHorse10\\\",1),(\\\"00000000000000000SeaHorse100\\\",1),(\\\"00000000000000000SeaHorse101\\\",1),(\\\"00000000000000000SeaHorse102\\\",1),(\\\"00000000000000000SeaHorse103\\\",1),(\\\"00000000000000000SeaHorse104\\\",1),(\\\"00000000000000000SeaHorse105\\\",1),(\\\"00000000000000000SeaHorse106\\\",1),(\\\"00000000000000000SeaHorse107\\\",1),(\\\"00000000000000000SeaHorse108\\\",1),(\\\"00000000000000000SeaHorse109\\\",1),(\\\"00000000000000000SeaHorse11\\\",1),(\\\"00000000000000000SeaHorse110\\\",1),(\\\"00000000000000000SeaHorse111\\\",1),(\\\"00000000000000000SeaHorse112\\\",1),(\\\"00000000000000000SeaHorse113\\\",1),(\\\"00000000000000000SeaHorse114\\\",1),(\\\"00000000000000000SeaHorse115\\\",1),(\\\"00000000000000000SeaHorse116\\\",1),(\\\"00000000000000000SeaHorse117\\\",1),(\\\"00000000000000000SeaHorse118\\\",1),(\\\"00000000000000000SeaHorse119\\\",1),(\\\"00000000000000000SeaHorse12\\\",1),(\\\"00000000000000000SeaHorse120\\\",1),(\\\"00000000000000000SeaHorse121\\\",1),(\\\"00000000000000000SeaHorse122\\\",1),(\\\"00000000000000000SeaHorse123\\\",1),(\\\"00000000000000000SeaHorse124\\\",1),(\\\"00000000000000000SeaHorse125\\\",1),(\\\"00000000000000000SeaHorse126\\\",1),(\\\"00000000000000000SeaHorse127\\\",1),(\\\"00000000000000000SeaHorse128\\\",1),(\\\"00000000000000000SeaHorse129\\\",1),(\\\"00000000000000000SeaHorse13\\\",1),(\\\"00000000000000000SeaHorse130\\\",1),(\\\"00000000000000000SeaHorse131\\\",1),(\\\"00000000000000000SeaHorse132\\\",1),(\\\"00000000000000000SeaHorse133\\\",1),(\\\"00000000000000000SeaHorse14\\\",1),(\\\"00000000000000000SeaHorse15\\\",1),(\\\"00000000000000000SeaHorse16\\\",1),(\\\"00000000000000000SeaHorse17\\\",1),(\\\"00000000000000000SeaHorse18\\\",1),(\\\"00000000000000000SeaHorse19\\\",1),(\\\"00000000000000000SeaHorse2\\\",1),(\\\"00000000000000000SeaHorse20\\\",1),(\\\"00000000000000000SeaHorse21\\\",1),(\\\"00000000000000000SeaHorse22\\\",1),(\\\"00000000000000000SeaHorse23\\\",1),(\\\"00000000000000000SeaHorse24\\\",1),(\\\"00000000000000000SeaHorse25\\\",1),(\\\"00000000000000000SeaHorse26\\\",1),(\\\"00000000000000000SeaHorse27\\\",1),(\\\"00000000000000000SeaHorse28\\\",1),(\\\"00000000000000000SeaHorse29\\\",1),(\\\"00000000000000000SeaHorse3\\\",1),(\\\"00000000000000000SeaHorse30\\\",1),(\\\"00000000000000000SeaHorse31\\\",1),(\\\"00000000000000000SeaHorse32\\\",1),(\\\"00000000000000000SeaHorse33\\\",1),(\\\"00000000000000000SeaHorse34\\\",1),(\\\"00000000000000000SeaHorse35\\\",1),(\\\"00000000000000000SeaHorse36\\\",1),(\\\"00000000000000000SeaHorse37\\\",1),(\\\"00000000000000000SeaHorse38\\\",1),(\\\"00000000000000000SeaHorse39\\\",1),(\\\"00000000000000000SeaHorse4\\\",1),(\\\"00000000000000000SeaHorse40\\\",1),(\\\"00000000000000000SeaHorse41\\\",1),(\\\"00000000000000000SeaHorse42\\\",1),(\\\"00000000000000000SeaHorse43\\\",1),(\\\"00000000000000000SeaHorse44\\\",1),(\\\"00000000000000000SeaHorse45\\\",1),(\\\"00000000000000000SeaHorse46\\\",1),(\\\"00000000000000000SeaHorse47\\\",1),(\\\"00000000000000000SeaHorse48\\\",1),(\\\"00000000000000000SeaHorse49\\\",1),(\\\"00000000000000000SeaHorse5\\\",1),(\\\"00000000000000000SeaHorse50\\\",1),(\\\"00000000000000000SeaHorse51\\\",1),(\\\"00000000000000000SeaHorse52\\\",1),(\\\"00000000000000000SeaHorse53\\\",1),(\\\"00000000000000000SeaHorse54\\\",1),(\\\"00000000000000000SeaHorse55\\\",1),(\\\"00000000000000000SeaHorse56\\\",1),(\\\"00000000000000000SeaHorse57\\\",1),(\\\"00000000000000000SeaHorse58\\\",1),(\\\"00000000000000000SeaHorse59\\\",1),(\\\"00000000000000000SeaHorse6\\\",1),(\\\"00000000000000000SeaHorse60\\\",1),(\\\"00000000000000000SeaHorse61\\\",1),(\\\"00000000000000000SeaHorse62\\\",1),(\\\"00000000000000000SeaHorse63\\\",1),(\\\"00000000000000000SeaHorse64\\\",1),(\\\"00000000000000000SeaHorse65\\\",1),(\\\"00000000000000000SeaHorse66\\\",1),(\\\"00000000000000000SeaHorse67\\\",1),(\\\"00000000000000000SeaHorse68\\\",1),(\\\"00000000000000000SeaHorse69\\\",1),(\\\"00000000000000000SeaHorse7\\\",1),(\\\"00000000000000000SeaHorse70\\\",1),(\\\"00000000000000000SeaHorse71\\\",1),(\\\"00000000000000000SeaHorse72\\\",1),(\\\"00000000000000000SeaHorse73\\\",1),(\\\"00000000000000000SeaHorse74\\\",1),(\\\"00000000000000000SeaHorse75\\\",1),(\\\"00000000000000000SeaHorse76\\\",1),(\\\"00000000000000000SeaHorse77\\\",1),(\\\"00000000000000000SeaHorse78\\\",1),(\\\"00000000000000000SeaHorse79\\\",1),(\\\"00000000000000000SeaHorse8\\\",1),(\\\"00000000000000000SeaHorse80\\\",1),(\\\"00000000000000000SeaHorse81\\\",1),(\\\"00000000000000000SeaHorse82\\\",1),(\\\"00000000000000000SeaHorse83\\\",1),(\\\"00000000000000000SeaHorse84\\\",1),(\\\"00000000000000000SeaHorse85\\\",1),(\\\"00000000000000000SeaHorse86\\\",1),(\\\"00000000000000000SeaHorse87\\\",1),(\\\"00000000000000000SeaHorse88\\\",1),(\\\"00000000000000000SeaHorse89\\\",1),(\\\"00000000000000000SeaHorse9\\\",1),(\\\"00000000000000000SeaHorse90\\\",1),(\\\"00000000000000000SeaHorse91\\\",1),(\\\"00000000000000000SeaHorse92\\\",1),(\\\"00000000000000000SeaHorse93\\\",1),(\\\"00000000000000000SeaHorse94\\\",1),(\\\"00000000000000000SeaHorse95\\\",1),(\\\"00000000000000000SeaHorse96\\\",1),(\\\"00000000000000000SeaHorse97\\\",1),(\\\"00000000000000000SeaHorse98\\\",1),(\\\"00000000000000000SeaHorse99\\\",1)])]))])))]}))))\"}")

  To rerun use: --match "/API Specifications/SHELLEY_TRANSACTIONS/TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=67)/"

  /Users/anviking/IOHK/cardano-wallet/lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs:747:13:
  2) API Specifications, SHELLEY_TRANSACTIONS, TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=68)
       expected: Status {statusCode = 202, statusMessage = "Accepted"}
        but got: Status {statusCode = 500, statusMessage = "Internal Server Error"}

       from the following response: Left (DecodeFailure "{\"code\":\"created_invalid_transaction\",\"message\":\"That's embarrassing. It looks like I've created an invalid transaction that could not be parsed by the node. Here's an error message that may help with debugging: HardForkApplyTxErrFromEra S (S (S (Z (WrapApplyTxErr {unwrapApplyTxErr = ApplyTxError [LedgerFailure (UtxowFailure (UtxoFailure (OutputTooBigUTxO [(Addr Mainnet (KeyHashObj (KeyHash \\\"7ae290756d9b7b19e3c3620e45e13598d2998b9c8f7019d6aa3ec8b9\\\")) (StakeRefBase (KeyHashObj (KeyHash \\\"8defec29958c443bad8af705c2460baaa86621a544b761e867d728fc\\\"))),Value 1997159861 (fromList [(PolicyID {policyID = ScriptHash \\\"4ff049585c4b3070563966370f5427d4a2f3588bce4146d57a93c7d3\\\"},fromList [(\\\"00000000000000000SeaHorse10\\\",1),(\\\"00000000000000000SeaHorse100\\\",1),(\\\"00000000000000000SeaHorse101\\\",1),(\\\"00000000000000000SeaHorse102\\\",1),(\\\"00000000000000000SeaHorse103\\\",1),(\\\"00000000000000000SeaHorse104\\\",1),(\\\"00000000000000000SeaHorse105\\\",1),(\\\"00000000000000000SeaHorse106\\\",1),(\\\"00000000000000000SeaHorse107\\\",1),(\\\"00000000000000000SeaHorse108\\\",1),(\\\"00000000000000000SeaHorse109\\\",1),(\\\"00000000000000000SeaHorse11\\\",1),(\\\"00000000000000000SeaHorse110\\\",1),(\\\"00000000000000000SeaHorse111\\\",1),(\\\"00000000000000000SeaHorse112\\\",1),(\\\"00000000000000000SeaHorse113\\\",1),(\\\"00000000000000000SeaHorse114\\\",1),(\\\"00000000000000000SeaHorse115\\\",1),(\\\"00000000000000000SeaHorse116\\\",1),(\\\"00000000000000000SeaHorse117\\\",1),(\\\"00000000000000000SeaHorse118\\\",1),(\\\"00000000000000000SeaHorse119\\\",1),(\\\"00000000000000000SeaHorse12\\\",1),(\\\"00000000000000000SeaHorse120\\\",1),(\\\"00000000000000000SeaHorse121\\\",1),(\\\"00000000000000000SeaHorse122\\\",1),(\\\"00000000000000000SeaHorse123\\\",1),(\\\"00000000000000000SeaHorse124\\\",1),(\\\"00000000000000000SeaHorse125\\\",1),(\\\"00000000000000000SeaHorse126\\\",1),(\\\"00000000000000000SeaHorse127\\\",1),(\\\"00000000000000000SeaHorse128\\\",1),(\\\"00000000000000000SeaHorse129\\\",1),(\\\"00000000000000000SeaHorse13\\\",1),(\\\"00000000000000000SeaHorse130\\\",1),(\\\"00000000000000000SeaHorse131\\\",1),(\\\"00000000000000000SeaHorse132\\\",1),(\\\"00000000000000000SeaHorse133\\\",1),(\\\"00000000000000000SeaHorse134\\\",1),(\\\"00000000000000000SeaHorse135\\\",1),(\\\"00000000000000000SeaHorse14\\\",1),(\\\"00000000000000000SeaHorse15\\\",1),(\\\"00000000000000000SeaHorse16\\\",1),(\\\"00000000000000000SeaHorse17\\\",1),(\\\"00000000000000000SeaHorse18\\\",1),(\\\"00000000000000000SeaHorse19\\\",1),(\\\"00000000000000000SeaHorse2\\\",1),(\\\"00000000000000000SeaHorse20\\\",1),(\\\"00000000000000000SeaHorse21\\\",1),(\\\"00000000000000000SeaHorse22\\\",1),(\\\"00000000000000000SeaHorse23\\\",1),(\\\"00000000000000000SeaHorse24\\\",1),(\\\"00000000000000000SeaHorse25\\\",1),(\\\"00000000000000000SeaHorse26\\\",1),(\\\"00000000000000000SeaHorse27\\\",1),(\\\"00000000000000000SeaHorse28\\\",1),(\\\"00000000000000000SeaHorse29\\\",1),(\\\"00000000000000000SeaHorse3\\\",1),(\\\"00000000000000000SeaHorse30\\\",1),(\\\"00000000000000000SeaHorse31\\\",1),(\\\"00000000000000000SeaHorse32\\\",1),(\\\"00000000000000000SeaHorse33\\\",1),(\\\"00000000000000000SeaHorse34\\\",1),(\\\"00000000000000000SeaHorse35\\\",1),(\\\"00000000000000000SeaHorse36\\\",1),(\\\"00000000000000000SeaHorse37\\\",1),(\\\"00000000000000000SeaHorse38\\\",1),(\\\"00000000000000000SeaHorse39\\\",1),(\\\"00000000000000000SeaHorse4\\\",1),(\\\"00000000000000000SeaHorse40\\\",1),(\\\"00000000000000000SeaHorse41\\\",1),(\\\"00000000000000000SeaHorse42\\\",1),(\\\"00000000000000000SeaHorse43\\\",1),(\\\"00000000000000000SeaHorse44\\\",1),(\\\"00000000000000000SeaHorse45\\\",1),(\\\"00000000000000000SeaHorse46\\\",1),(\\\"00000000000000000SeaHorse47\\\",1),(\\\"00000000000000000SeaHorse48\\\",1),(\\\"00000000000000000SeaHorse49\\\",1),(\\\"00000000000000000SeaHorse5\\\",1),(\\\"00000000000000000SeaHorse50\\\",1),(\\\"00000000000000000SeaHorse51\\\",1),(\\\"00000000000000000SeaHorse52\\\",1),(\\\"00000000000000000SeaHorse53\\\",1),(\\\"00000000000000000SeaHorse54\\\",1),(\\\"00000000000000000SeaHorse55\\\",1),(\\\"00000000000000000SeaHorse56\\\",1),(\\\"00000000000000000SeaHorse57\\\",1),(\\\"00000000000000000SeaHorse58\\\",1),(\\\"00000000000000000SeaHorse59\\\",1),(\\\"00000000000000000SeaHorse6\\\",1),(\\\"00000000000000000SeaHorse60\\\",1),(\\\"00000000000000000SeaHorse61\\\",1),(\\\"00000000000000000SeaHorse62\\\",1),(\\\"00000000000000000SeaHorse63\\\",1),(\\\"00000000000000000SeaHorse64\\\",1),(\\\"00000000000000000SeaHorse65\\\",1),(\\\"00000000000000000SeaHorse66\\\",1),(\\\"00000000000000000SeaHorse67\\\",1),(\\\"00000000000000000SeaHorse68\\\",1),(\\\"00000000000000000SeaHorse69\\\",1),(\\\"00000000000000000SeaHorse7\\\",1),(\\\"00000000000000000SeaHorse70\\\",1),(\\\"00000000000000000SeaHorse71\\\",1),(\\\"00000000000000000SeaHorse72\\\",1),(\\\"00000000000000000SeaHorse73\\\",1),(\\\"00000000000000000SeaHorse74\\\",1),(\\\"00000000000000000SeaHorse75\\\",1),(\\\"00000000000000000SeaHorse76\\\",1),(\\\"00000000000000000SeaHorse77\\\",1),(\\\"00000000000000000SeaHorse78\\\",1),(\\\"00000000000000000SeaHorse79\\\",1),(\\\"00000000000000000SeaHorse8\\\",1),(\\\"00000000000000000SeaHorse80\\\",1),(\\\"00000000000000000SeaHorse81\\\",1),(\\\"00000000000000000SeaHorse82\\\",1),(\\\"00000000000000000SeaHorse83\\\",1),(\\\"00000000000000000SeaHorse84\\\",1),(\\\"00000000000000000SeaHorse85\\\",1),(\\\"00000000000000000SeaHorse86\\\",1),(\\\"00000000000000000SeaHorse87\\\",1),(\\\"00000000000000000SeaHorse88\\\",1),(\\\"00000000000000000SeaHorse89\\\",1),(\\\"00000000000000000SeaHorse9\\\",1),(\\\"00000000000000000SeaHorse90\\\",1),(\\\"00000000000000000SeaHorse91\\\",1),(\\\"00000000000000000SeaHorse92\\\",1),(\\\"00000000000000000SeaHorse93\\\",1),(\\\"00000000000000000SeaHorse94\\\",1),(\\\"00000000000000000SeaHorse95\\\",1),(\\\"00000000000000000SeaHorse96\\\",1),(\\\"00000000000000000SeaHorse97\\\",1),(\\\"00000000000000000SeaHorse98\\\",1),(\\\"00000000000000000SeaHorse99\\\",1)])]))])))]}))))\"}")

  To rerun use: --match "/API Specifications/SHELLEY_TRANSACTIONS/TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=68)/"

Randomized with seed 301508993

Finished in 41.1069 seconds
7 examples, 2 failures
```
First we succeed (no need to split change)
Then we fail (we should have split change, but we didn't)
Then we succeed (we should and do split the change)

Reply:
And without this PR / rebased against master:
```
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=64)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=65)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=66)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=67) FAILED [1]
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=68) FAILED [2]
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=69) FAILED [3]
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=70) FAILED [4]
```
so it seems to make sense

Reply:
Hi @Anviking 

Many thanks for running these tests!

I've added a further commit 4c36492a6a9c5067ce25e6ad6aacc266cfa2960e which I believe fixes the problem you've highlighted.

I think your assessment here is totally correct:
> I think this is due to `assignCoinsToChangeMaps` adding `Coin` values to the bundle after we do the size check.

To fix this problem, we now perform size assessment based on what the size **_would be_** if the change map were paired with the **_maximum possible_** ada quantity:

```hs
assessBundleSizeWithMaxCoin
    :: TokenBundle -> TokenBundleSizeAssessment
assessBundleSizeWithMaxCoin =
    assessBundleSize . flip TokenBundle.setCoin (maxBound @Coin)
```
    
Of course, this approach has a (very small) chance of over-splitting a change map that would be marginally under the limit when combined its final ada quantity, but this would happen only very occasionally. I think this approach is certainly preferable to creating change bundles that are slightly over the limit, which would lead to transactions being rejected (as you have seen in testing).

Reply:
Just for the record: I've rested this by rebasing @Anviking 's branch on top of this PR's branch, and the integration tests now pass: 🎉 

```hs
API Specifications
  SHELLEY_TRANSACTIONS
[cardano-wallet.integration:Notice:221] [2021-03-09 09:08:23.68 UTC] Configuring cluster in /tmp/test1945
[cardano-wallet.integration:Notice:228] [2021-03-09 09:08:25.97 UTC] Not registering any stake pools due to NO_POOLS=1. Some tests may fail.
[cardano-wallet.integration:Notice:255] [2021-03-09 09:08:26.15 UTC] Setting up faucet...
[cardano-wallet.integration:Notice:684] [2021-03-09 09:08:30.73 UTC] Wallet url: http://127.0.0.1:36019/
EKG url: none
Prometheus url: none
SMASH url: http://localhost:45375/

    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=64)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=65)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=66)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=67)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=68)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=69)
    TRANS_ASSETS_CREATE_02c - Send SeaHorses (nPerAddr=70)

Finished in 44.8178 seconds
7 examples, 0 failures
```